ui/src/landing/CreateWish.js

```diff
@@ -13,8 +13,9 @@ export default class CreateWish extends Component {
     }
   }
 
-  validFields = (age) => ({
-      validAge: age !== '' && (Number(age) < 18 && Number(age) >= 2)
+  validFields = (age, name) => ({
+      validAge: age !== '' && (Number(age) < 18 && Number(age) >= 2),
+      validName: (name !== '') && (name.match(/[\w\-’\s]+/)) ? true : false
```

Comment:
This could potentially be split into its own request, but I believe this regex will not allow for diacritics in names. Let me know what you think.

Reply:
No worries I can add it in this request. Yeah, that's true, I'll revise the regex to take into account names with diacritics.